recipe/bld.bat

```diff
@@ -1,6 +1,8 @@
 if not exist %PREFIX% mkdir %PREFIX%
+if errorlevel 1 exit /b 1
```

Comment:
Would there be an `errorlevel` if we haven't run anything? Think that may be why [CI is failing]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=842734&view=logs&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=6119ccbe-9301-594f-7c27-f792b80a7fcc ) (as it doesn't seem to be pointing to this line. In particular we are seeing (note `BUILD_PREFIX` is not referenced above and may also already exist):

```
(base) %SRC_DIR%>if not exist %BUILD_PREFIX% mkdir %BUILD_PREFIX% 

(base) %SRC_DIR%>if errorlevel 1 exit /b 1 
```

If we do want this, we may want to add it as part of the `if` right after `mkdir`

```suggestion
if not exist %PREFIX% (
  mkdir %PREFIX%
  errorlevel 1 exit /b 1
)
````